content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md

```diff
@@ -290,7 +290,8 @@ After this setting is applied, the JWT will contain the updated `iss` value. In
 
 ### Customizing the subject claims for an organization or repository
 
-To help improve security, compliance, and standardization, you can customize the standard claims to suit your required access conditions. If your cloud provider supports conditions on subject claims, you can create a condition that checks whether the `sub` value matches the path of the reusable workflow, such as `"job_workflow_ref: "octo-org/octo-automation/.github/workflows/oidc.yml@refs/heads/main""`. The exact format will vary depending on your cloud provider's OIDC configuration. To configure the matching condition on {% data variables.product.prodname_dotcom %}, you can can use the REST API to require that the `sub` claim must always include a specific custom claim, such as `job_workflow_ref`. You can use the [OIDC REST API](/rest/actions/oidc) to apply a customization template for the OIDC subject claim; for example, you can require that the `sub` claim within the OIDC token must always include a specific custom claim, such as `job_workflow_ref`.
+To help improve security, compliance, and standardization, you can customize the standard claims to suit your 
```

Comment:
What is the change suggested here?

Reply:
@N-Usha it's mainly just formatting around some of the monospaced element:

> `"job_workflow_ref:octo-org/octo-automation/.github/workflows/oidc.yml@refs/heads/main"`

It's got a little bit broken in the diff because of a newline, I think, which has obscured the relevant change in the diff.

@afairb 👋 would you mind editing this slightly? It won't matter for the render in the site but it would be ideal if we could do. Then we can get this merged!

Reply:
Sorry, I hadn't spotted this earlier. Crept in by accident and removed now :)